### ISSUE TITLE
Remove Shops with Admin Permission?

### ISSUE THREAD

**Issue by Skyfay** [Patbox/UniversalShops#37]
Is there a way to remove the store from people other than the admin? It doesn't work with OP and I don't see any authorization for it.

**Comment by AmaratheCapybara** [comment]
I think you have to be in creative mode.

**Comment by BasToTheMax** [comment]
yeah, i'm wondering about this as well. Seems like i can get rid of it using `/setblock` command, but surely that's not the correct way?
